lib/stores/GCSDataStore.ts

```diff
@@ -175,22 +155,15 @@ class GCSDataStore extends DataStore {
     })
   }
 
-  /**
-   * Get file metadata from the GCS Object.
-   *
-   * @param  {string} file_id     name of the file
-   * @return {object}
-   */
-  // @ts-expect-error TS(2416): Property 'getOffset' in type 'GCSDataStore' is not... Remove this comment to see the full error message
-  getOffset(file_id: any) {
+  getOffset(file_id: string): Promise<File> {
     return new Promise((resolve, reject) => {
       if (!file_id) {
         reject(ERRORS.FILE_NOT_FOUND)
         return
       }
 
-      const file = this.bucket.file(file_id)
-      file.getMetadata((error: any, metadata: any) => {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      this.bucket.file(file_id).getMetadata((error: any, metadata: any) => {
```

Comment:
Are you still working on this type or is this intended to be left as-is?

Reply:
`error` and `metadata` are both `any` in the underlying types, but TS complains so I make it explicit too.